pages/docs/how-to-guides/testimonials.mdx

```diff
@@ -14,7 +14,7 @@ You can give a Testimonial for another LinkFree user by going to their Profile a
 The issue will look like this:
 
 ![LinkFree testimonial issue](https://user-images.githubusercontent.com/624760/210427249-25848dd6-463a-4748-aa60-45c6e53f49a8.png)
-
+**NOTE**: Don't have a Linkfree account but want to give a testimonial? No worries! Log in to your Github account (if you don't have one, check out [Eddie's course](https://www.eddiejaoude.io/course-github-profile-landing) to help you) and follow the above steps.
```

Comment:
Non-blocking: I think these two should be capitalized.
```suggestion
**NOTE**: Don't have a LinkFree account but want to give a testimonial? No worries! Log in to your GitHub account (if you don't have one, check out [Eddie's course](https://www.eddiejaoude.io/course-github-profile-landing) to help you) and follow the steps above.
```

Reply:
I agree 👍 I think I can accept your suggestions to the PR